R/pkg/inst/extdata/spark_download.csv

```diff
@@ -0,0 +1,2 @@
+"url","default"
+"http://apache.osuosl.org",TRUE
```

Comment:
I wouldn't recommend hardcoding or relying on certain mirrors - apache mirrors are sort of dynamic - it's better to pull from 
http://www.apache.org/dyn/closer.cgi?as_json=1
like
http://www.apache.org/dyn/closer.lua/spark/spark-1.6.2/spark-1.6.2-bin-hadoop2.6.tgz